delocate/tests/test_wheelies.py

```diff
@@ -342,3 +343,22 @@ def ignore_libextfunc2(path: str) -> bool:
         assert delocate_wheel(
             RPATH_WHEEL, 'tmp.whl', lib_filt_func=ignore_libextfunc2
         ) == stray_libs_only_direct
+
+
+def test_fix_toplevel() -> None:
+    # Test wheels which are not organized into packages.
+
+    with InTemporaryDirectory():
+        # The module was set to expect its dependency in the libs/ directory
+        os.symlink(DATA_PATH, 'libs')
```

Comment:
I think this won't work as expected on Windows, should we ever get there : https://docs.python.org/3/library/os.html

Reply:
I knew it was Unix specific, and I've already been using symlinks for the rpath tests.

Adding support for Window's DLL's to delocate would just be writing an entire new library on top of the existing one.  I don't really see much code that would be shared between them since DLL's are more different to dylibs than Linux so's are, and those were already handled well by auditwheel.

Switching symlink out for something more cross-platform is still a reasonable idea, so I'll do that.